### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by joao-borges** [appium/java-client#1796] [General]
Hello guys, just wondering when the next release will come out. We need one feature included in selenium 4.5 and I saw you guys already updated it.

Thanks

**Comment by saikrishna321** [comment]
> Hello guys, just wondering when the next release will come out. We need one feature included in selenium 4.5 and I saw you guys already updated it.
> 
> 
> 
> Thanks

Will be done today.

**Reply by joao-borges** [reply]
Hi @saikrishna321 I don't see the release.

**Reply by anil-gr1** [reply]
Hi, When can we expect the next release?

**Reply by isitrunning** [reply]
Hi, I see [8.2.1 release notes](https://github.com/appium/java-client/commit/6a1429437a193aacbc19a63b4c588605bc8f9c23), but couldn't find the actual 8.2.1 release yet on maven central. 
Am I doing something wrong, or it's not released yet? 
Thank you!
